pom.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+    http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.openmicroscopy</groupId>
+  <artifactId>ome-jai</artifactId>
```

Comment:
For Fiji, I think the major conflicting issue was associated with the package name i.e. `com.sun.media...` rather than the artifact name per se. That being said, if we agree to rename the artifactId, starting at a low version number makes sense.

NB: we might also need to consider the implications of http://forum.imagej.net/t/fiji-and-bioformats-conflicting-jars/5736 which highlights some potential issues arising with renamed JARs distributed via the various Fiji update sites.